README.md

```diff
@@ -76,8 +76,25 @@ async function quickstart(
       branchName,
     },
   };
-  await cb.runBuildTrigger(request);
+  const [resp] = await cb.runBuildTrigger(request);
   console.info(`triggered build for ${triggerId}`);
+  const [build] = await resp.promise();
+
+  const STATUS_LOOKUP = [
```

Comment:
So, this is actually coming from protos. We don't export it now (subject to change! need to think about it), but you can do

```
const protos = require(path.join(path.dirname(require.resolve('@google-cloud/cloudbuild')), '..', 'protos', 'protos'));
console.log(protos.google.devtools.cloudbuild.v1.Build.Status);
```

to get this:
```
{
  STATUS_UNKNOWN: 0,
  QUEUED: 1,
  WORKING: 2,
  SUCCESS: 3,
  FAILURE: 4,
  INTERNAL_ERROR: 5,
  TIMEOUT: 6,
  CANCELLED: 7
}
```

Reply:
this is for samples, so I'd rather not leak that abstraction ... I also don't want to just print the #, so I'm thinking the look up table might be best for now.